http/1.6/modules/ROOT/pages/http-listener-ref.adoc

```diff
@@ -210,93 +333,135 @@ blue
 
-In the following example, an endpoint is defined to store logos for an account:
+* *Body*
+* *Headers*
+* *Status code*
+* *Reason phrase*
 
-. When the storage is successful, a `201 Created` is returned with a body, such as `Corporate Logo has been stored as a MuleSoft logo`.
-. When the storage fails, the status code is defined through the `errorCode` variable (if available) or `500` by default.
-. A custom header is added (`X-Time`) as well.
-. A body, such as `Corporate Logo could not be stored`, is returned.
-. If there is a `CONNECTIVITY` error storing the logo, for example, the returned status code would be `504`, while any other errors would result in a `500`.
+=== Create Sending HTTP Responses Mule App Example
 
-NOTE: While a reason phrase is not defined for error responses, the connector attempts to define one based on the status code. Thus, a `Gateway Timeout` or `Internal Server Error` is returned in the scenarios explained before.
+The following example shows how to configure HTTP responses for the HTTP *Listener* source in Studio. The example is a Mule app flow where an endpoint stores logos for an account:
 
-[source,xml,linenums]
+* When the storage is successful, an HTTP regular response returns with status code `201`, reason phrase `Created`, and body as `Corporate Logo has been stored as a MuleSoft logo`.
+* When the storage fails, an HTTP error response returns where the status code is defined through the `errorCode` variable (if available) or a `500` code by default.
+** The custom header returns `X-Time`.
+** The body returns `Corporate Logo could not be stored`.
+** If there is a `CONNECTIVITY` error storing the logo, the returned status code is `504`, while any other errors result in a `500` status code.
+
+While a reason phrase is not defined for error responses, HTTP Connector attempts to define a reason phrase based on the status code. Thereby, a `Gateway Timeout` or `Internal Server Error` error can return for the previous explained HTTP responses scenarios.
+
+To create the Mule app in Studio, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP>Listener*.
+. Drag *Listener* to the Studio canvas.
+. Set *Path* to `/account/{accountId}/logo`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Listener* in the app.
+. Set *Host* to `0.0.0.0` and *Port* to `8081`.
+. Click *OK*.
+. In the HTTP *Listener* configuration screen, click the *Responses* tab.
+. In the *Response* section, set these values:
++
+* *Body* : `output text/plain --- vars.logoName ++ ' has been stored as a ' ++ vars.accountId ++ ' logo.'`
+* *Status code* : `201`
+* *Reason phrase* : `Created`
+
+[start=9]
+. In the *Error Response* section, set these values:
++
+* *Body* : `vars.logoName ++ ' could not be stored.'`
+* *Headers* :
++
+[source,DataWeave,linenums]
```

Comment:
I wonder if it would be more easily understood to show the task first and the response explanations after?